### PR TITLE
added the detailed view for a single activity

### PR DESCRIPTION
**Pull request by dylan17th**
# Description
added a detailed view for the user for each activity.
Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [X] New feature (non-breaking change which adds functionality)

## Change Status

- [X] Complete, but not tested (may need new tests)

# Checklist

- [X] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [X] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] There are no merge conflicts

### CODE REVIEW

**Comment by michaelharms6010** [comment]
I'm a little puzzled by the conflicts here - are you up to date with master?

File: my-school/src/Components/AcitvityOverview.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
+import React, { useEffect, useState } from 'react';
```
Comment (michaelharms6010) [inline my-school/src/Components/AcitvityOverview.js:1]: There is a typo in this filename. Please fix prior to merge

Comment (dylan17th) [reply my-school/src/Components/AcitvityOverview.js]: just fixed it

Comment (michaelharms6010) [reply my-school/src/Components/AcitvityOverview.js]: Thank u sir

**Pull request merged**